components/mention/mention.component.ts

```diff
@@ -118,13 +120,16 @@ export type MentionPlacement = 'top' | 'bottom';
     @if (hasFeedback && !!status) {
       <nz-form-item-feedback-icon class="ant-mentions-suffix" [status]="status" />
     }
+    @if (nzAllowClear && hasValue()) {
+      <nz-select-clear class="ant-mentions-suffix" [clearIcon]="nzClearIcon" (clear)="clear()" />
```

Comment:
<img width="542" height="111" alt="Image" src="https://github.com/user-attachments/assets/91eb8b31-fc6e-457c-b103-e29f698ffa65" />

Due to the different DOM structure, nz-select-clear cannot be used directly.

Reply:
done :)